src/app/layout/qgslayoutimageexportoptionsdialog.h

```diff
@@ -0,0 +1,143 @@
+/***************************************************************************
+                         qgslayoutimageexportoptionsdialog.h
+                         -------------------------------------
+    begin                : December 2017
+    copyright            : (C) 2017 by Nyall Dawson
+    email                : nyall dot dawson at gmail dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSLAYOUTIMAGEEXPORTOPTIONSDIALOG_H
+#define QGSLAYOUTIMAGEEXPORTOPTIONSDIALOG_H
+
+#include <QDialog>
+#include "ui_qgslayoutimageexportoptions.h"
+
+
+/**
+ * A dialog for customising the properties of an exported image file.
```

Comment:
`\since QGIS 3.0`?

Reply:
Not sure here. It's in app, so is the version information really useful for anything? Happy to add it if you think there's value.